projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -74,6 +74,20 @@ public function get_preferred_view( $screen, $fallback_global_preference = true
 		return parent::get_preferred_view( $screen, $fallback_global_preference );
 	}
 
+	/**
+	 * Point the Site Editor's `< Dashboard` link to wpcom home.
+	 *
+	 * Although this isn't strictly an admin menu item, it belongs here because it's part of
+	 * changing wp-admin links to their wp.com equivalents.
+	 *
+	 * @param  array $settings Editor settings.
+	 * @return array           Updated Editor settings.
+	 */
+	public function site_editor_dashboard_link( $settings ) {
+		$settings['__experimentalDashboardLink'] = 'https://wordpress.com/home/' . $this->domain;
```

Comment:
Normally I'd suggest doing something like this here, just in case the value contains something weird.
```suggestion
		$settings['__experimentalDashboardLink'] = 'https://wordpress.com/home/' . rawurlencode( $this->domain );
```
But it looks like there's a ton of other code in this file already not doing that, and the value is even being munged to replace `/` characters (probably for exactly that lack), so doing it only here would be inconsistent and might even break something depending on the unencoded behavior.

Reply:
Yeah keeping consistent is usually better than being "right", but good to notice 👍